es.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/elastic/go-elasticsearch/v7"
 )
 
+type Client = elasticsearch.Client
```

Comment:
what for?

Reply:
it's only boilerplate to avoid import. I'll remove it.